with-legend-state-supabase/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "expo-supa-legend-state",
+  "version": "1.0.0",
+  "main": "expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo run:android",
+    "ios": "expo run:ios",
+    "web": "expo start --web"
+  },
+  "dependencies": {
+    "@expo/metro-runtime": "~3.2.1",
+    "@legendapp/state": "^3.0.0-alpha.37",
+    "@supabase/supabase-js": "^2.45.1",
+    "@types/react": "~18.2.79",
+    "@types/uuid": "^10.0.0",
```

Comment:
Can we move these dependencies and the `typescript` dependency to `devDepenedencies` since they are only required during development?

Reply:
good catch, thanks